app/services/github_user_service.rb

```diff
@@ -8,4 +8,26 @@ def find_or_create(github_user_params)
       name: github_user_params.name
     ).find_or_create_by!(gh_id: github_user_params.id)
   end
+
+  # Performs a request to retrieve all teams associated
+  # to the github username given by `github_login`.
+  # Teams' visiblity depends on authentication privileges
+  # in `octokit_client`.
+  def fetch_teams_for_user(github_login, octokit_client)
+    organizations_logins =
+      octokit_client
+        .organizations(github_login)
+        .map do |organization|
```

Comment:
Align .map with octokit_client on line 18.<br>Pass &:login as an argument to map instead of a block.